### ISSUE TITLE
[Bug] Chisel blocks disappear around entangled blocks

### ISSUE THREAD

**Issue by P3rf3ctXZer0** [SuperMartijn642/Entangled1.12#21]
**Version Info**
- Minecraft 1.12.2
- Forge Final
- entangled-1.3.3-mc1.12.jar

**Description of the Bug**
SEE VIDEO LINK

**Steps to Reproduce**
SEE VIDEO LINK

**Video**
https://youtu.be/KOUk2cSyHP8 My channel is 18+ so I don't deal with youtube hell just ignore that or use the dropbox link and download it. https://www.dropbox.com/s/gyqr4w8fbw8aa7m/2021-06-07%2020-24-56.mkv?dl=1

**Comment by P3rf3ctXZer0** [comment]
Please note to get to the point where the bug happens 100% of the time I spent 8 hours of free testing for you so do your part and figure out exactly why this is occurring :D It does not happen with tesseract just entangled.

**Comment by SuperMartijn642** [comment]
What is the modpack you're playing on?
```
Please note to get to the point where the bug happens 100% of the time I spent 8 hours of free testing for you so do your part and figure out exactly why this is occurring
```
Just so you know, I also spent hundreds of hours developing mods for free 😉

**Comment by P3rf3ctXZer0** [comment]
Oh my modpack Fiction Friction my bad I thought I put it up ... also forget my smart ass comment I was super drained and annoyed it is not your mod that annoyed me it was the steps to repeat it.

https://www.curseforge.com/minecraft/modpacks/fiction-friction

Let me know if I can help I have history of finding obscure bugs but todays specifically I am not working since yesterday I spent the entire day and evening working.

**Comment by SuperMartijn642** [comment]
No worries, I know how frustrating it can be to pin down inconsistent bugs.
I will probably have time to do some testing later this week or early next week.

**Comment by SuperMartijn642** [comment]
I reproduce the issue with just Entangled, Chisel and their dependencies.

**Comment by SuperMartijn642** [comment]
I fixed it now in version 1.0.8.

Thank you for reporting the issue!

**Comment by P3rf3ctXZer0** [comment]
No problem glad I can help :D

**Comment by P3rf3ctXZer0** [comment]
Is the update posted? I keep looking for it.

**Comment by SuperMartijn642** [comment]
Whoops, I meant version 1.3.4.
1.0.8 was the Core lib version 😅

**Comment by P3rf3ctXZer0** [comment]
https://i.imgur.com/TmmcfAV.png 

But its not available!

**Comment by P3rf3ctXZer0** [comment]
![TmmcfAV 1](https://user-images.githubusercontent.com/10351221/122365054-32487300-cf0f-11eb-83e3-b22120797ec4.png)

But it does not exist on curse

**Comment by P3rf3ctXZer0** [comment]
![flYmLY2 1](https://user-images.githubusercontent.com/10351221/122365450-894e4800-cf0f-11eb-8097-fe2cdca5c360.png)

**Comment by SuperMartijn642** [comment]
I fixed the dependencies now in version 1.3.4a.

Thanks again!
